### PR TITLE
[14.0][MIG] website_mass_mailing_name: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT32118

### CODE REVIEW

**Comment by ernestotejeda** [comment]
I fixed the test in mail_layout_preview that is blcking this PR: https://github.com/OCA/social/pull/831
ping @pedrobaeza

**Comment by ernestotejeda** [comment]
ping @pedrobaeza @Tardo

**Comment by pedrobaeza** [comment]
But this is 14.0, not 13.0...

**Review by pedrobaeza — APPROVED** [review]
Functionally tested on runbot:

- Login.
- Add the mailing subscription block.
- Go to anonymous browser.
- Subscribe putting the name.
- Get the mailing contact with the email and name.

File: website_mass_mailing_name/demo/assets.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (Tardo) [inline website_mass_mailing_name/demo/assets.xml]: Is this file in use? i don't see "demo" key in the manifest...

File: website_mass_mailing_name/demo/assets.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<!-- Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+<template id="assets_frontend_demo" inherit_id="website.assets_frontend">
+    <xpath expr=".">
+        <script
+                type="text/javascript"
```
Comment (Tardo) [inline website_mass_mailing_name/demo/assets.xml]: Is this a user tour or a test tour? Is it possible to make user tours for website?

**Comment by ernestotejeda** [comment]
Thanks @tardo, I moved that code to https://github.com/OCA/social/pull/830/files#diff-7240970f5b3561896e0b77811b9fd3c4cfc54f6d9bd167358a6a926bf169005cR19 and i forgot to remove this file

**Comment by ernestotejeda** [comment]
@Tardo change done

**Review by Tardo — APPROVED** [review]

**Pull request merged**